pkgs/development/libraries/jsoncpp/default.nix

```
@@ -30,8 +30,14 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake python ];
```

```suggestion
  nativeBuildInputs = [ cmake python validatePkgConfig ];
```
It'd be nice to make use of the pkg-config validation hook from https://github.com/NixOS/nixpkgs/pull/87866/files

---

I added it, but unfortunately it is not able to detect the error that this package had.